circuits/rollup.circom

```diff
@@ -254,8 +257,9 @@ template Rollup(nTx, nLevels) {
     for (i = 0; i < 16; i++){
         checkFees.accFee[i] <== Tx[nTx-1].accFeeOut[i];
     }
-
     checkFees.feeTotals <== feeTotals;
 
     checkFees.feeTotalOk === 1;
+
+    // privEthAddress === pubEthAddress;
```

Comment:
is this an intended comment?

Reply:
Yes, since this constraint will become real in the next feature. I decided to split features in two pull-request, but I already had the code written :) so I commented it

Reply:
ofc, just checking ^^